Jenkinsfile

```diff
@@ -0,0 +1,81 @@
+pipeline {
+    agent any
+
+    environment {
+        VM_NAME = "gitubuntu-ci-${currentBuild.getNumber()}"
+        UACLIENT_BEHAVE_CONTRACT_TOKEN = credentials('ua-contract-token')
```

Comment:
I think we need the staging token as well

Reply:
+1 just added it